tests/baselines/reference/arrowFunctionParsingDoesNotConfuseParenthesizedObjectForArrowHead.js

```diff
@@ -18,7 +18,7 @@ const test = () => ({
 //// [arrowFunctionParsingDoesNotConfuseParenthesizedObjectForArrowHead.js]
 var test = function () { return ({
     // "Identifier expected." error on "!" and two "Duplicate identifier '(Missing)'." errors on space.
-    prop: !value,
+    prop: !value, // remove ! to see that errors will be gone

```

Comment:
Rather than modifying unrelated tests, can you just add new tests to show that your PR fixes the bug?

It's okay to accept a new baseline for an old test if its output changes (that is very normal), but it's not normal to add comments to old, unrelated test cases to show that an emit bug is fixed.

Reply:
@jakebailey You're right, and I didn't modify other test cases to confirm this fix. The comments were already there in the original `.ts` files (but were omitted from the reference baselines.)

As I mentioned, I also added a single test case, `commentsOnObjectLiteral5.ts`, to just confirm this fix.

Reply:
Oh shoot, you're totally right. I didn't notice these weren't the original TS files!

Reply:
For example, this is the original (intact) test case file for the line you commented on (i.e.,  `tests/cases/compiler/arrowFunctionParsingDoesNotConfuseParenthesizedObjectForArrowHead.ts`):

```ts
// regression test for https://github.com/microsoft/TypeScript/issues/32914
declare var value: boolean;
declare var a: any;

const test = () => ({
    // "Identifier expected." error on "!" and two "Duplicate identifier '(Missing)'." errors on space.
    prop: !value, // remove ! to see that errors will be gone
    run: () => { //replace arrow function with regular function to see that errors will be gone
        // comment next line or remove "()" to see that errors will be gone
        if(!a.b()) { return 'special'; }

        return 'default';
    }
}); 
```